r/R/compute.R

```diff
@@ -34,6 +48,30 @@ call_function <- function(function_name, ..., args = list(...), options = empty_
   compute__CallFunction(function_name, args, options)
 }
 
+#' List available Arrow C++ compute functions
+#'
+#' This function lists the names of all available Arrow compute functions.
+#' These can be called by passing to [call_function()], or they can be
+#' called by name with an `arrow_` prefix inside a `dplyr` verb.
```

Comment:
I assume some of these would not actually work because the options are not wired up in `make_compute_options`. Do you think it's worth addressing that?

Reply:
Good call.